src/application.py

```diff
@@ -100,9 +95,9 @@ def __init__(self, application_id, **kwargs):
         operation_action.connect("activate", actions.perform_operation, self)
         self.add_action(operation_action)
 
+        actions_settings = settings.get_child("actions")
         for action_key in ["center", "smoothen"]:
-            action = settings.get_child("actions").create_action(action_key)
-            self.add_action(action)
+            self.add_action(actions_settings.create_action(action_key))
```

Comment:
Not sure why this got reverted, as I put it like this already in commit 961d75a38dd89d7fc24cc95d75d054a786e56f7a.